server/services/diskUsageService.js

```diff
@@ -19,7 +19,7 @@ const filterMountPoint =
 
 const diskUsage = {
   linux: () =>
-    execFile('df --block-size=1024 --portability | tail -n+2', {
+    execFile('df | tail -n+2', {
```

Comment:
I think the problem was that apline does not support any of the long names, perhaps we can use `-k -P` instead?

I think `-P` is healthy so that we're sure the columns are in the same order.
I think explicit block-size `-k` could be useful too if some system uses single byte blocksize or otherwise.

Reply:
I won't explain as long as in #823 but I think the posix option just standardized the column naming, not the columns order. I also think that all linux will have the same default block size, it may change only on different platform like *BDS or MacOS but any the script only handle linux and mac for now.
I think the options would be useless but if it safer we can use `df -kPl` and so have teh same command used for `['darwin', 'linux'];` simplifying the code. Someone just need to confim that `-P` is also working on Mac.

Reply:
Had no issues running on MacOS 10.11:
```
➜  ~ df -kPl
Filesystem                        1024-blocks      Used Available Capacity  Mounted on
/dev/disk0s2                        487712924 342782424 144674500    71%    /
localhost:/yxD52ojDj7YEU08xiCrBrk   487712924 487712924         0   100%    /Volumes/MobileBackups
```